bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -761,10 +761,18 @@ void DWARFRewriter::updateDebugInfo() {
               : std::optional<std::string>(opts::DwarfOutputPath.c_str());
       std::string DWOName = DIEBlder.updateDWONameCompDir(
           *StrOffstsWriter, *StrWriter, *CU, DwarfOutputPath, std::nullopt);
+      auto DWODIEBuilderPtr = std::make_unique<DIEBuilder>(
+          BC, &(**SplitCU).getContext(), DebugNamesTable, CU);
+      DWODIEBuildersByCU.emplace_back(std::move(DWODIEBuilderPtr));
+      DIEBuilder &DWODIEBuilder = *DWODIEBuildersByCU.back().get();
```

Comment:
This could be just one line:
DIEBuilder &DWODIEBuilder = *DWODIEBuildersByCU.emplace_back(std::move(DWODIEBuilderPtr)).get();